CMakeLists.txt

```
@@ -323,7 +323,11 @@ If(Geant3_FOUND)
  add_subdirectory (geane)
  add_subdirectory (trackbase)
 EndIf ()
-add_subdirectory (eventdisplay)
+
+execute_process(COMMAND ${ROOTSYS}/bin/root-config --has-opengl OUTPUT_VARIABLE ROOT_HAS_OPENGL)
```

```suggestion
execute_process(COMMAND ${ROOT_CONFIG_EXECUTABLE} --has-opengl OUTPUT_VARIABLE ROOT_HAS_OPENGL)
```

Have not tried it, but this should be available, if I read our `cmake/FindROOT.cmake` correctly.

Once we switch to the ROOT CMake package, the `execute_process` can be dropped, because it populates a variable `${ROOT_opengl_FOUND}` which can be used directly as `if(ROOT_opengl_FOUND)`.

---

done. thanks